internal/webauthn/credential.go

```diff
@@ -0,0 +1,69 @@
+package webauthn
+
+import (
+	"github.com/go-webauthn/webauthn/webauthn"
+
+	"github.com/authelia/authelia/v4/internal/configuration/schema"
+	"github.com/authelia/authelia/v4/internal/model"
+)
+
+func VerifyCredential(config *schema.WebAuthn, credential *model.WebAuthnCredential, mds MetaDataProvider) (result VerifyCredentialResult) {
+	var (
+		c   *webauthn.Credential
+		err error
+	)
+
+	if len(credential.Attestation) == 0 {
+		result.MissingStatement = true
+	}
```

Comment:
Is it worth unit testing these expected cases?

Reply:
Yeah that's why I extracted this instead of leaving it in the commands package, done.